tests/snapshots/assets__check_file@packages-touying-core.typ-120.snap

```diff
@@ -872,14 +891,15 @@ snapshot_kind: text
     supplement: eqt.supplement,
     eval(
       "$" + result.sum(default: "") + "$",
-      scope: eqt.scope + (
-        cover: (..args) => {
-          let cover = eqt.scope.at("cover", default: cover)
-          if args.pos().len() != 0 {
-            cover(args.pos().first())
-          }
-        },
-      ),
+      scope: eqt.scope
+        + (
+          cover: (..args) => {
+            let cover = eqt.scope.at("cover", default: cover)
+            if args.pos().len() != 0 {
+              cover(args.pos().first())
+            }
+          },
+        ),
     ),
```

Comment:
original behavior is better. it put `(` on the same line with `+`. For short chains I this is better.

Reply:
a tricky case. i think we could not have a better result if we must put a closure here, if we do not allow hanging the last item